.github/CONTRIBUTING.md

```diff
@@ -25,6 +25,7 @@ All types of contributions are encouraged and valued. See the [Table of Contents
 - [Style Guides](#style-guides)
 - [Typst Code](#typst-code)
```

Comment:
In hindsight I should probably have indented these when I first created this guide.
```suggestion
  - [Typst Code](#typst-code)
  - [Commit Messages](#commit-messages)
  - [Branches](#branches)
```